standard/expressions.md

```diff
@@ -2817,17 +2817,28 @@ The `unchecked` operator is convenient when writing constants of the signed inte
 
 ### 11.7.19 Default value expressions
 
-A default value expression is used to obtain the default value ([§9.3](variables.md#93-default-values)) of a type. Typically a default value expression is used for type parameters, since it might not be known if the type parameter is a value type or a reference type. (No conversion exists from the `null` literal ([§6.4.5.7](lexical-structure.md#6457-the-null-literal)) to a type parameter unless the type parameter is known to be a reference type ([§8.2](types.md#82-reference-types)).)
+A default value expression is used to obtain the default value ([§9.3](variables.md#93-default-values)) of an explicit or inferred type. Typically, a default value expression is used for type parameters, since it might not be known if the type parameter is a value type or a reference type. (No conversion exists from the `null` literal ([§6.4.5.7](lexical-structure.md#6457-the-null-literal)) to a type parameter unless the type parameter is known to be a reference type ([§8.2](types.md#82-reference-types)).)
```

Comment:
Since the introduction of the default literal, I'm not sure that this "typically" is correct. For example, I often see it used for default parameter values, especially for CancellationToken:

```csharp
public void DoSomethingAsync(CancellationToken cancellationToken = default)
```

No type parameters involved. Would "One notable use" be a suitable substitution here? And should everything after the first sentence be in a note?

Reply:
I agree. It's not normative. §10.2.12 and §10.3.8 have normative text that there isn't a conversion from `null` to a type parameter that isn't known to be a reference type.

Reply:
I don't think the phrasing "an explicit or inferred type" is correct here (and elsewhere), we should follow the phraseology used for `null`. So something along the lines of "obtain the default value of a [target] type" depending on the context.

Reply:
I agree with Nigel's point on "inferred type". When I read that, I think it would allow this:

```csharp
var i = default;
```

That's not allowed. I think the following suggestion on "conversion to any type" would be better: it would work for both known types and type parameters.